examples/performance-sample/build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id 'net.saliman.properties' version '1.4.6'
-    id 'com.marklogic.ml-data-hub' version '4.0.1'
+    id 'com.marklogic.ml-data-hub' version '4.0.2'
 }

```

Comment:
Looks like version wasn't changed in this instance. Everything else looks good.

Reply:
Good catch, fixed